package.json

```diff
@@ -34,8 +34,8 @@
   },
   "devDependencies": {
     "axe-core": "3.5.5",
-    "cypress": "6.8.x",
-    "cypress-axe": "0.12.2",
+    "cypress": "7.5.0",
+    "eslint-plugin-cypress-dev": "3.0.0",
```

Comment:
This line is an artifact from the old base, it's not part of initial "test rebase commit". Should be removed.
Great work!